substrate-archive/src/actors/workers/blocks.rs

```diff
@@ -67,7 +67,7 @@ where
 {
 	pub fn new(ctx: ActorContext<B, D>, db: DatabaseAct<B>, meta: MetadataAct<B>) -> Self {
 		Self {
-			rt_cache: RuntimeVersionCache::new(ctx.backend.clone()),
+			rt_cache: Arc::new(RuntimeVersionCache::new(ctx.backend.clone())),
```

Comment:
@insipx @emostov `ArcSwap` does not `impl Clone` since v1.1 and [it seems like](https://github.com/vorner/arc-swap/issues/38) wrapping the thing in yet another `Arc` is an ok thing to do. We can do it like I did above or in `RuntimeVersionCache` itself. I went with this because it's a little less ugly than `Arc<ArcSwap<…>>`.

Wdyt?

Reply:
Yeahp, I think this would be the easiest way to do it